src/test/java/org/cactoos/list/BehavesAsList.java

```diff
@@ -65,12 +64,12 @@ public boolean matchesSafely(final List<E> list) {
         new Assertion<>(
             "must have an index for the sample item",
             list.indexOf(this.sample),
-            new MatcherOf<>(i -> i >= 0)
+            new Verifies<>(i -> i >= 0)
```

Comment:
@victornoel IMHO, I think having matchers `IsGreaterThan`, `IsLessThan` and 
`IsEmpty` for list and iterable in cactoos-matchers will be very nice :) We have only static methods of them  (greatThan and lessThan) in Hamcrest.

Reply:
@baudoliver7 that's a good idea, could you open tickets in cactoos-matchers for those? You will get points for them too! Maybe one for `IsGreaterThan`/`IsLessThan` and one for `IsEmpty`? we can discuss them there then!

Reply:
@victornoel Great !